platform/ethereum/blockbook/model.go

```diff
@@ -0,0 +1,77 @@
+package blockbook
+
+import "math/big"
+
+type Page struct {
+	Transactions []Transaction `json:"transactions,omitempty"`
+	Tokens       []Token       `json:"tokens,omitempty"`
+}
+
+type NodeInfo struct {
+	Blockbook *Blockbook `json:"blockbook"`
+	Backend   *Backend   `json:"backend"`
+}
+
+type Blockbook struct {
+	BestHeight int64 `json:"bestHeight"`
+}
+
+type Backend struct {
+	Blocks int64 `json:"blocks"`
+}
+
+type Block struct {
+	Transactions []Transaction `json:"txs"`
+}
+
+type Transaction struct {
+	TxID             string            `json:"txid"`
+	Vin              []Output          `json:"vin"`
+	Vout             []Output          `json:"vout"`
+	BlockHeight      uint64            `json:"blockHeight"`
+	BlockTime        int64             `json:"blockTime"`
+	Value            string            `json:"value"`
+	Fees             string            `json:"fees"`
+	TokenTransfers   []TokenTransfer   `json:"tokenTransfers,omitempty"`
+	EthereumSpecific *EthereumSpecific `json:"ethereumSpecific,omitempty"`
+}
+
+type Output struct {
+	Value     string   `json:"value,omitempty"`
+	Addresses []string `json:"addresses"`
+}
+
+// TokenType specifies type of token
+type TokenType string
+
+// ERC20TokenType is Ethereum ERC20 token
+const ERC20TokenType TokenType = "ERC20"
```

Comment:
can you reuse https://github.com/trustwallet/blockatlas/blob/3eeb1d248527e110a7b1a7ea08b965ff3c880b2e/pkg/blockatlas/tx.go#L18 ?

Reply:
done